src/AppBundle/Controller/PaController.php

```diff
@@ -18,6 +18,11 @@ class PaController extends AbstractController
      */
     public function indexAction(Request $request)
     {
+        $user = $this->getUser();
+        if ($user->getRoleName() == EntityDir\User::ROLE_PA && !$user->isAgreeTermsUse()) {
+            throw new \RuntimeException('The PA user didn not agree with terms and conditions');
```

Comment:
I'll remove the whole check. I've moved it before activation, so the user won't be even able to log and reach that page